google/resource_bigtable_instance.go

```diff
@@ -265,3 +312,77 @@ func expandBigtableClusters(clusters []interface{}, instanceID string) []bigtabl
 	}
 	return results
 }
+
+func resourceBigtableInstanceClusterReorderTypeList(diff *schema.ResourceDiff, meta interface{}) error {
+	keys := diff.GetChangedKeysPrefix("cluster")
+	if len(keys) == 0 {
+		return nil
+	}
+
+	oldCount, newCount := diff.GetChange("cluster.#")
+	var count int
+	if oldCount.(int) < newCount.(int) {
+		count = newCount.(int)
+	} else {
+		count = oldCount.(int)
+	}
+
+	// simulate Required:true and MinItems:1
+	if count < 1 {
+		return nil
```

Comment:
We should error out here, right? That way we don't continue if users have specified no clusters / haven't specified a cluster.

Reply:
yeah good call, will return an error here